docs/installation_linux.rst

```diff
@@ -1,41 +1,78 @@
 Install LISA on Linux
 =====================
 
-Minimum System Requirement
---------------------------
-1. Your favourite Linux distro supporting Python 3.8+
+Minimum System Requirements
+---------------------------
+
+1. Your favorite Linux distribution supporting Python 3.8+
 2. Dual core processor
 3. 4 GB system memory
 
-We will guide you through the installation of LISA on Linux. The following commands assume Ubuntu 18.04 as the underlying Linux distro.
+We will guide you through the installation of LISA on Linux.
+The following commands assume Ubuntu is being used.
+
 
 Install Python on Linux
 -----------------------
 
-LISA has been tested to work with `Python 3.8 64-bit
-<https://www.python.org/>`__. The latest version of Python 3 is
-recommended. If you find that LISA is not compatible with higher version
-Python, `please file an
-issue <https://github.com/microsoft/lisa/issues/new>`__.
+LISA has been tested to work with `Python 3.8 64-bit <https://www.python.org/>`__ and above.
+The latest version of Python 3 is recommended. If you find that LISA is not compatible
+with higher version Python, `please file an issue <https://github.com/microsoft/lisa/issues/new>`__.
+
+To check which version of Python is used on your system, run the following:
+
+.. code:: bash
+
+   python3 --version
+
+If you need to install a different Python package, there are likely packaged versions for
+your distro.
 
-Run following commands to install Python 3.8:
+Here is an example to install Python 3.9 on Ubuntu 20.04
```

Comment:
We haven't plan to upgrade the default version. But if it works, we can make 3.10 is default version for deployment, and make sure all places updated.

Reply:
What are all the places?

Reply:
> We can look at it in a follow-up PR, but best practice is to pin dependencies as little as possible. Pinning dependencies moves the contract of trust from the individual projects to us. For well-maintained projects, we really don't need to pin anything. For less stable projects we can likely reduce the places in the version number we care about, `=~1.2` vs `~=1.2.3`. Regardless, we can discuss further in a follow-up PR.

We can discuss further. It's not about the frequency of upgrading packages, it's about how we can release consistent behaviors. For example, we tested well with 1.3, and released a stable version. But the dependency may install 1.3.1 after one week.
We learned a lot from relying on latest version, including a recent Sev1 ICM. If we release an open-source package, the user manages their release cycle, but we are responsible for keeping LISA stable for all downstream teams. We're asked many times, what's our release model, and how can we keep things stable.

Reply:
There are some different strategies. Let's continue the conversation somewhere else.
Updated the versions, so resolving this comment.